integration-tests/test-cases/scenario-tests/src/test/java/io/cellery/integration/scenario/tests/petstore/domain/PetAccessory.java

```diff
@@ -0,0 +1,54 @@
+/*
+ *  Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+package io.cellery.integration.scenario.tests.petstore.domain;
+
+/**
+ * This includes the functions related to pet-store scenario.
+ */
+public class PetAccessory {
+    private int amount;
+    private String xpath;
+
+    /**
+     *
+     * @param amount
+     *        The number of items of pet accessory.
```

Comment:
Formatting issue. Please fix this and others.

Reply:
Updated.